gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -301,6 +301,10 @@ handlers:
   oauth2:
     response:
       strict: false
+  scim:
```

Comment:
is this exposed in the helm chart as well?

Reply:
The **handlers** section is fully configurable in Helm's values YAML. Therefore, a **scim** section can be added there, resulting in its presence in the gravitee.yaml file.